DEVELOPER.md

```diff
@@ -0,0 +1,178 @@
+- [GitHub repo](https://github.com/bcgov/rsbc-digital-forms/) (source code)
+- [OpenShift project (licence plate 'be78d6')](https://console.apps.silver.devops.gov.bc.ca/k8s/cluster/project.openshift.io~v1~Project) environments:
+    - [DEV](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/be78d6-dev/core~v1~Service)
+    - [TEST](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/be78d6-test/core~v1~Service)
+    - [PROD](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/be78d6-prod/core~v1~Service)
+
+
+## Set up a local development environment
+
+These steps walk you through creating your own local development environment on Windows 10. The process will be similar for Mac or Linux. If you are using a Mac or Linux system to set up a development environment, please document any extra steps you needed to get up and running, then update this message to say that you tested the steps on your platform.
+
+
+### Requirements
+
+You need the following tools installed before you can create a development environment. In August 2023, these were the versions available:
+
+- **git** --version > git version 2.40.1.windows.1
+- **node** --version > v18.16.0
+- **npm** --version > 9.6.7
+- **python** --version > Python 3.10.6
+- **pip** --version > pip 22.3 from C:\Python310\lib\site-packages\pip (python 3.10)
+- **choco** --version > 1.1.0
+- **Docker Desktop**, which requires Hyper-V to be installed. Hyper-V requires a CPU with hardware virtualization and a non-Home edition of Windows (e.g. Windows Pro or Enterprise edition). You may be able to use the standard Docker on Windows Home editions with WSL2 and no Hyper-V, but that configuration has not been tested.
+
+The typical integrated development environment used on this project is Visual Studio Code.
+
+
+### Installing Docker Desktop
+
+To install Docker Desktop with Chocolatey, run the following command from an elevated command prompt:
+
+    choco install docker-desktop docker-cli docker-compose
+
+Docker Desktop will appear on  the Start menu, but you need to log in and out to start the services before using it. Also, docker-compose and docker.exe will be on your path. In August 2023, these were the versions used:
+
+- **Docker Desktop** version 4.21.0, which included:
+  - **docker** --version > 24.0.2, build cb74dfc
+  - **docker-compose** --version > v2.19.1
+
+
+### Access to Artifactory
+
+Once Docker is installed, you must log in to the container registry from Docker. The container images are stored in an OpenShift Artifactory repository. The command to log in is shown below. You can obtain a working credential from a member of the development team and replace the word "PASSWORD" below:
+
+    docker login --username default-be78d6-kqpjvc --password PASSWORD artifacts.developer.gov.bc.ca/dbe7-images
```

Comment:
Similar to the Password, remove the actual username too. We dont need the username to be public too.